frontend/payment-klarna/store/getters.ts

```diff
@@ -71,6 +73,14 @@ export const getters: GetterTree<CheckoutState, RootState> = {
     const external_payment_methods = config.klarna.external_payment_methods ? config.klarna.external_payment_methods.map(mapRedirectUrl) : null;
     const external_checkouts = config.klarna.external_checkouts ? config.klarna.external_checkouts : null;
 
+    //translate
+    _.find(config.klarna.options.additional_checkboxes, function(o) {
```

Comment:
What is `o`?  Can you give it a better name?

I don't think `find` is the right function to use here, maybe `forEach`?